node-util/sbin/oo-last-access

```diff
@@ -27,8 +27,14 @@ class LastAccessUpdater
     @last_access_data = Hash.new
     @addrs_ignore = Array.new
     @hosts_ignore = ["localhost"]
+    @useragent_ignore = []
   end
 
+  def load_useragent_ignore
+    ignore_list = OpenShift::Config.new.get("UNIDLER_IGNORE_LIST") || ""
```

Comment:
Instead of using `|| ""`, you can specify a default value to `get`: `OpenShift::Config.new.get("UNIDLER_IGNORE_LIST", "")`